docs/integrations.md

```diff
@@ -2,14 +2,16 @@
 
 ## VS Code (Official)
 
-Download the [Ruff VS Code extension](https://marketplace.visualstudio.com/items?itemName=charliermarsh.ruff),
+Download
+the [Ruff VS Code extension](https://marketplace.visualstudio.com/items?itemName=charliermarsh.ruff),
```

Comment:
Is this the IntelliJ thing where it bumps the word _before_ the link to the next line for unknown reasons?

Reply:
Can we revert the formatting changes?

Reply:
Oh why did it do this. I disabled markdown formatting more than once because it's supper intrusive. Sorry for that